packages/sync/integration_test/test/offline.test.js

```diff
@@ -24,7 +24,7 @@ const newClient = async (clientOptions = {}) => {
   return await createClient(config);
 };
 
-describe('AeroGear Apollo GraphQL Voyager client', function() {
+describe('AeroGear Apollo GraphQL Voyager client', function () {
```

Comment:
Formatter issue?

Reply:
I think my beautify used the linter rules for the project as a guideline so I committed it.